apps/mobile/src/components/widgets/accounts/accounts-widget.tsx

```diff
@@ -53,21 +52,18 @@ export function AccountsWidget({ accounts, wallets }: AccountsWidgetProps) {
               sheetRef={sheetRef}
               sheet={<AccountSelectorSheet sheetRef={sheetRef} />}
             />
-            {hasWallets && <Balance balance={balance} variant="heading03" />}
+            {hasWallets && <Balance balance={totalBalance} variant="heading03" />}
           </Box>
         }
       >
+        {/* FIXME  stop using mock accounts and use the real ones 
```

Comment:
I'm still using `mockAccounts` as I need to update the actual accounts types. 

I am extending `AccountStore` to add extra data we need

```
export interface MockedAccount extends AccountStore {
  type: WalletStore['type'];
  balance: Money;
  fingerprint: string;
  accountIndex: number;
}
```